build.gradle

```diff
@@ -250,6 +254,8 @@ dependencies {
     testFunctionalCompile("com.fasterxml.jackson.core:jackson-databind:2.9.8") {
         force = true
     }
+    testFunctionalCompileOnly 'org.projectlombok:lombok:1.18.8'
```

Comment:
same here , would it be better to pull this lombok version rather than hardcoded in all places